kaggle-classification/keras_trainer/model.py

```diff
@@ -113,6 +115,7 @@ def train(self, train):
 
   def predict(self, texts):
     data = self._prep_texts(texts)
+    self._load_model()
```

Comment:
Why is this necessary? The model i loaded when the class is instantiated and then loaded whenever the model is trained.

Reply:
Good point. Removed.